atomic_reactor/core.py

```diff
@@ -230,12 +278,14 @@ def __init__(self, base_url=None, timeout=120, **kwargs):
         if hasattr(docker, 'AutoVersionClient'):
             client_kwargs['version'] = 'auto'
 
-        try:
-            # docker-py 2.x
-            self.d = docker.APIClient(**client_kwargs)
-        except AttributeError:
-            # docker-py 1.x
-            self.d = docker.Client(**client_kwargs)
+        self.retry_times = retry_times
+        if not str(retry_times).isdigit() or retry_times == 0:
```

Comment:
But let's say it's `"4"`. The isdigit() test will succeed so we won't change it. Then we'll pass a string straight though... won't that break?